### PR TITLE
Prepare folder structure for forward benchmarks

### PR DESCRIPTION
**Pull request by tgymnich**
a branch rename closed #543 ...

### CODE REVIEW

File: enzyme/benchmarks/ReverseMode/ba/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Run regression and unit tests
-add_lit_testsuite(bench-matdescent "Running enzyme benchmarks tests"
+add_lit_testsuite(bench-ba-reverse "Running enzyme benchmarks tests"
```
Comment (wsmoses) [inline enzyme/benchmarks/ReverseMode/ba/CMakeLists.txt:2]: It still seems like the git move was messed up?

Comment (tgymnich) [reply enzyme/benchmarks/ReverseMode/ba/CMakeLists.txt:2]: its gonna be messed up again if we squash and merge this PR. Maybe we can merge it without squashing this time...

Comment (wsmoses) [reply enzyme/benchmarks/ReverseMode/ba/CMakeLists.txt:2]: Sure, though this particular file seems weird since its a move from matdescent to ba

Comment (wsmoses) [reply enzyme/benchmarks/ReverseMode/ba/CMakeLists.txt:2]: Oh, huh it looks like the commit itself is fine

**Review by wsmoses — APPROVED** [review]

**Pull request merged**